optuna/samplers/_tpe/_truncnorm.py

```diff
@@ -207,10 +207,10 @@ def rvs(
     b: np.ndarray,
     loc: Union[np.ndarray, float] = 0,
     scale: Union[np.ndarray, float] = 1,
-    size: int = 1,
```

Comment:
[Notes] I confirmed that `size` of the `rvs` function was not used in the feature branch.